rfcs/text/0000-bundle-types.md

```diff
@@ -0,0 +1,90 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC: Bundle Types
+
+## Summary
+
+Formalizes data bundle types, definitions, target users/consumers, and specific use cases to improve clarity and
+confidence among DCP developers when working with the DCP data model.
+
+## Author(s)
+
+* [Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)
+* [Brian Hannafious](mailto:bhannafi@ucsc.edu)
+* [Hannes Schmidt](mailto:hannes@ucsc.edu)
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+There are currently no documented definitions for what a "bundle" is in the HCA DCP, including no definitions of what
+they are, what they contain, or who they support. Lack of clarity causes confusion for DCP developers and data consumers
+about how the data and metadata in the DCP are structured and organized.
+
+### User Stories
+
+1. As a tool developer, I would like to identify and get raw data in the DSS so that I can run my data processing and
+   analysis tools.
+
+1. As a tool developer, I would like to identify and get alignment results and count matrices in order to produce
+   expression matrices.
+
+1. As a computational biologist, I would like to get expression matrices for the Immune Cell Atlas project so that I can
+   do my research.
+
+1. As an external user of the DSS who is developing analysis tools, I would like to browse, query, and get raw and
+   processed data based on some rough criteria so that I can understand what the data are and develop my tool.
+
+1. As an Azul or Query Service developer I want to be able to distinguish between different types of bundles in order to
+   know which bundle to index, link, etc. or to provide bundle types to users as a search criterion.
+
+1. As a computational biologist, I would like to know where the reference data is.
```

Comment:
I think a more accurate Users Story would be "As a computational biologist, I would like to use the same references that the HCA DCP uses in their analysis". And if I understand the current state, the 'reference bundle' is not accessible to the user, and therefore, this is not fully resolved by this RFC?

Reply:
@jahilton thank you for the re-formulated user story, I adopted your wording.

In the current state, as far as I know reference data is not stored in the DCP at all and is fetched from external sources by the pipelines.

The implication of this RFC is that the reference bundle format would be adopted to store references within the DCP DSS. These bundles *would* be accessible to the user.